### PR TITLE
[WIP] - Added documentation for the following plugins

### PR DESCRIPTION
**Pull request by Rick-Ritense**
- Object token authentication
- Objecten API
- Objecttypen API

The Objecten API frontend hasn't been created yet so it is missing the configuration example image

The Objecten API and Objecttypen API page should be extended with the REST api's that are available after those have been written

### CODE REVIEW

File: using-valtimo/plugin/object-token-authentication/configure-plugin.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+The Object Token Auhentication plugin is used to provide authentication for ZGW (Zaakgericht Werken) plugins that connect to APIs which 
+use a token as authentication. Currently the plugins that this plugin can provide authentication for are the following:
+
+- [Objecten API](../objecten-api/configure-plugin.md)
+- [Objecttypen API](../objecttypen-api/configure-plugin.md)
+
+### Prerequisites
+
+This page requires you to have knowledge on how to configure Objecten API
```
Comment (marijnritense) [inline using-valtimo/plugin/object-token-authentication/configure-plugin.md:11]: This page requires knowledge on how to configure Objecten API. For more information, see [here](whateverthelocationis).

**Comment by marijnritense** [comment]
I don't think the filenames should all be configure-plugin.md. Each should be more specific, as this will also show up in the path. So it would be configure-objecten-api-plugin, etc.

File: using-valtimo/plugin/object-token-authentication/configure-plugin.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+## Configuring the plugin
+
+A plugin configuration is required before the plugin can be used. A general description on how to configure
+plugins can be found [here](../configure-plugin.md).
+
+If the Object Token Auhentication plugin is not visible in the plugin menu, it is possible the application is missing a dependency.
+Instructions on how to add the Object Token Auhentication dependency can be found 
+[here](../../../valtimo-implementation/modules/objecttokenauthentication.md).
```
Comment (marijnritense) [inline using-valtimo/plugin/object-token-authentication/configure-plugin.md:20]: Can also replace the ../../../ to /

File: using-valtimo/plugin/object-token-authentication/configure-plugin.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+To configure this plugin the following properties have to be entered:
+- **Token** The token is configured in the Objects administration page 'api authorizations'. The token should have permissions 
+to object(types) registered to it
```
Comment (marijnritense) [inline using-valtimo/plugin/object-token-authentication/configure-plugin.md:24]: Missing a . at the end.

File: using-valtimo/plugin/object-token-authentication/configure-plugin.md (hunk 1 above)
Comment (ivo-ritense) [reply using-valtimo/plugin/object-token-authentication/configure-plugin.md]: This might be difficult. We don't have a page with more information.

**Review by mritense — APPROVED** [review]

**Pull request merged**